src/lib/select/select.spec.ts

```diff
@@ -1701,6 +1701,12 @@ describe('MdSelect', () => {
         expect(select.getAttribute('aria-labelledby')).toBe('myLabelId');
       });
 
+      it('should not have aria-labelledby in the DOM if it`s not specified', () => {
+        fixture.detectChanges();
+
+        expect(select.getAttribute('aria-labelledby')).toBe(null);
```

Comment:
You can switch this one to `select.hasAttribute('aria-labelledby')` to make it a bit more clear.

Reply:
Fixed